tests/api/test_dynamo.py

```diff
@@ -3,6 +3,64 @@
 from hyp3_api import dynamo
 
 
+def test_count_jobs(tables):
+    table_items = [
+        {
+            'job_id': 'job1',
+            'user_id': 'user1',
+            'status_code': 'status1',
+            'request_time': '2000-01-01T00:00:00+00:00',
+        },
+        {
+            'job_id': 'job2',
+            'user_id': 'user1',
+            'status_code': 'status1',
+            'request_time': '2000-01-01T00:00:00+00:00',
+        },
+        {
+            'job_id': 'job3',
+            'user_id': 'user2',
+            'status_code': 'status1',
+            'request_time': '2000-01-01T00:00:00+00:00',
+        },
+    ]
+    for item in table_items:
+        tables['jobs_table'].put_item(Item=item)
+
+    assert dynamo.count_jobs('user1') == 2
+    assert dynamo.count_jobs('user2') == 1
+
+
+def test_count_jobs_by_start(tables):
+    table_items = [
+        {
+            'job_id': 'job1',
+            'user_id': 'user1',
+            'status_code': 'status1',
+            'request_time': '2000-01-01T00:00:00+00:00',
+        },
+        {
+            'job_id': 'job2',
+            'user_id': 'user1',
+            'status_code': 'status1',
+            'request_time': '2000-01-02T00:00:00+00:00',
+        },
+        {
+            'job_id': 'job3',
+            'user_id': 'user1',
+            'status_code': 'status1',
+            'request_time': '2000-01-03T00:00:00+00:00',
+        },
+    ]
+    for item in table_items:
+        tables['jobs_table'].put_item(Item=item)
+
+    start = '2000-01-01T00:00:00z'
```

Comment:
`+00:00` instead of `z`?
might make start/end `01:T01` and `02T23` and assert we only get a single job back?  or add one or two more calls/asserts